public/app/core/components/TraceToLogs/TagMappingInput.tsx

```diff
@@ -1,26 +1,16 @@
 import { css } from '@emotion/css';
 import React from 'react';
 
-import { GrafanaTheme2, KeyValue } from '@grafana/data';
+import { GrafanaTheme2 } from '@grafana/data';
 import { SegmentInput, useStyles2, InlineLabel, Icon } from '@grafana/ui';
 
-const EQ_WIDTH = 3; // = 24px in inline label
-
 interface Props {
-  values: Array<KeyValue<string>>;
-  onChange: (values: Array<KeyValue<string>>) => void;
+  values: Array<{ key: string; value?: string }>;
```

Comment:
the `KeyValue` is just `Record<string, any>` which isn't helpful type at all. I assume the name was a bit misleading.